cypress/support/gutenberg/commands/controls.js

```diff
@@ -412,6 +415,100 @@ function radioControl( name, value, options = {} ) {
 		.click( { force: true } )
 }
 
+/**
+ * Command for adjusting the form token input.
+ *
+ * @param {string} name
+ * @param {*} value
+ * @param {Object} options
+ */
+function formTokenControl( name, value, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+		parentSelector,
+		supportedDelimiter = [],
+	} = options
+
+	const selector = () => cy.getBaseControl( name, {
+		isInPopover,
+		parentSelector,
+		supportedDelimiter,
+	} )
+
+	const isWpLessThan58 = compareVersions( Cypress.env( 'WORDPRESS_VERSION' ), '5.8.0', '<' )
+
+	beforeAdjust( name, value, options )
+	value.forEach( val => {
+		if ( isWpLessThan58 ) {
+		// The suggestions list does not display on WP versions less than 5.8. Type and enter the values.
+			selector()
+				.find( '.components-form-token-field__input' )
+				.click( { force: true } )
+				.type( val, { force: true } )
+		} else {
+			selector()
+				.find( '.components-form-token-field__input' )
+				.click( { force: true } )
+		}
+		selector()
+			.find( `ul.components-form-token-field__suggestions-list li:contains(${ val })` )
+			.click( { force: true } )
+	} )
+}
+
+/**
+ * Command for changing the style of the block.
+ *
+ * @param {string} value
+ */
+export function adjustBlockStyle( value = '' ) {
+	cy.get( '.block-editor-block-styles' )
+		.find( `div[aria-label="${ value }"]` )
+		.click( { force: true } )
+}
+
+/**
+ * Command for adjusting the checkbox control
+ *
+ * @param {string} name
+ * @param {boolean} value
+ * @param {Object} options
+ */
+function checkboxControl( name, value = true, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+		parentSelector,
+		supportedDelimiter = [],
+	} = options
+
+	const selector = name => cy
+		.getBaseControl( name, {
+			isInPopover, parentSelector, supportedDelimiter,
+		} )
+		.find( `.components-base-control__field:contains(${ name })` )
```

Comment:
Can we use `containsRegExp` here?

Reply:
tests are failing if we use `.contains( containsRegExp( name ) )`. We need to yield `.components-base-control__field`